src/components/POLine/Item/ItemView.js

```diff
@@ -15,123 +15,120 @@ import {
   ProductIdDetails,
 } from '@folio/stripes-acq-components';
 
-class ItemView extends Component {
-  static propTypes = {
-    poLineDetails: PropTypes.object,
-  }
+const ItemView = ({ poLineDetails }) => {
+  const instanceId = get(poLineDetails, 'instanceId');
+  const title = get(poLineDetails, 'titleOrPackage');
+  const titleValue = instanceId
+    ? <Link to={`/inventory/view/${instanceId}`}>{title}</Link>
+    : title;
+  const contributors = get(poLineDetails, 'contributors', []);
 
-  static defaultProps = {
-    poLineDetails: {},
-  }
+  return (
+    <Fragment>
```

Comment:
shouldn't it be `<>`?

Reply:
https://folio-project.slack.com/archives/C210UCHQ9/p1579893578027700?thread_ts=1579888944.023400&cid=C210UCHQ9

Reply:
short syntax 👍